conda/meta.yaml

```diff
@@ -1,22 +1,27 @@
+{% set name = "zstash" %}
+{% set version = "0.4.0" %}
+
 package:
-    name: zstash
-    version: 0.4.0
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    git_url: https://github.com/E3SM-Project/zstash
-    git_rev: v0.4.0
+  git_url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9fce7561efa0d512567f89219a1f1642210810523a550b1974a17fdba1b6f544
 
 build:
-    script: python setup.py install
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
This is the standard install command on conda-forge, better than `python setup.py install`.  Ask or Google if you want details on why.